index.html

```diff
@@ -1222,6 +1316,27 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            User agents SHOULD re-fetch and process the manifest on every page
+            load. When the processing is successful, user agents MAY apply updated
+            manifest to any current and future <a>application contexts</a>
+            associated with the application. When the user agent fetches a
+            manifest as part of an update check, it MUST [=url/equal|compare=] the fetched
+            processed manifest's [=manifest/identity=] with the identity of the
+            app being updated. If it is not equal, the user agent MUST NOT
+            update the installed application's manifest. 
+          </p>
+          <section>
+            <h3>
+              Privacy and security consideration
+            </h3>
+            <p>
+              User agents SHOULD be careful about performing update to manifest
```

Comment:
I think we need to just fold this into the section above. We probably don't need to call it out.... in fact, I think an update is really just a "re-install", so we might be able to leverage what's already in the installation section.

Reply:
It seems a very different UX from re-install as it's performed without user action. I agree the above section is very hand-wavy, and the privacy/security section warns about what NOT to do (update identity related information without user consent). And leave the rest to individual implementation. How should I describe this better? Or I just move this as the next paragraph to:
```
User agents SHOULD re-fetch and process the manifest on every page load.
When the processing is successful, user agents MAY apply updated manifest
to any current and future application contexts associated with the application.
```
?

Reply:
> It seems a very different UX from re-install as it's performed without user action. 

Agree. 

>  How should I describe this better? 

I think what you stated in this paragraph is good (in as far that it does highlight what the risks are, and what browsers MAY choose to do about it). So yeah, if you move this up, I think that will give us a good foundation to build on.

Reply:
> It seems a very different UX from re-install as it's performed without user action.

On second thought, I'm unsure if all UAs will do this without user action (although, implemented in multiple OSs... like Chrome OS and Windows). I feel like this might still be an area where we might see some variation on how different implementations handle this. 

In any case, I think the text you've proposed covers the privacy/security aspects that are of most concern with respect to upgrades.